Jenkinsfile

```diff
@@ -260,15 +261,19 @@ void defaultInit() {
 // + other customizations
 void applyCustomizations() {
   libName = 'node'
+  // always want to do this
+  createNpmrc()
   bumpVersion = { isDevRelease ->
-    // Get the dependencies
-    sh 'npm ci --no-audit'
-    // Get the target version with no build meta
-    newVersion = getNewVersion(isDevRelease, false)
-    // Update to the new version, not tagging for dev releases
-    sh "npm version ${isDevRelease ? '--no-git-tag-version' : '-m "Update version -> %s"'} ${newVersion}"
-    // Set env variable version from updated package.json
-    env.NEW_SDK_VERSION = sh returnStdout: true, script: 'jq -j .version package.json'
+    withNpmEnv("ARTIFACTORY_DOWN", registryArtifactoryDown) {
```

Comment:
Given this appears to work I guess `registryArtifactoryDown` automatically uses the getter `getRegistryArtifactoryDown()`.

Reply:
yes it's a groovy-ism